README.md

```diff
@@ -1 +1,226 @@
 # origin-metrics
+
+TODO: write brief introduction here.
+
+## Building the Docker Containers
+
+The docker containers can be built using the `hack/build-images.sh` script.
+
+	cd hack
+	./build-images.sh --prefix=openshift/origin- --version=devel
+
+Where the `--prefix` value is used to specify a prefix to the docker image and the `--version` value is used to specify the version tag for the resulting docker images.
+
+For instance, as with the above command, a prefix of `openshift/origin-` and version `devel` will create the following docker images:
+
+	openshift/origin-metrics-deployer
+	openshift/origin-metrics-heapster
+	openshift/origin-metrics-cassandra
+	openshift/origin-metrics-hawkular-metrics
+	
+## Setting up your System
+
+The Metrics components require a properly running OpenShift instances. How to properly install and configure OpenShift is beyond the scope of this document. Please see the [OpenShift documentation](https://docs.openshift.org/latest/welcome/index.html) on how to properly install and setup your system.
+
+### Enabling the Read Only Kubelet Endpoint
+
+Currently the kubelet endpoints are secured with certificate authentication and are not accessible to metric contianers. This is a known issue which is currently being worked on: https://github.com/openshift/origin/pull/4873 
+
+Until this is properly fix in OpenShift, you will need to enable the RO endpoint for each of your nodes.
+
+For each of your node's `node-config.yaml` you will need to run the following command to enable read only access:
+
+	cat >> node-config.yaml << DONE
+	kubeletArguments:
+ 	read-only-port:
+ 	- "10266"
+	DONE
+
+### Creating Persistent Storage
```

Comment:
We're going to deploy with emptydir volumes. Then the user can use `oc volume` to change those to real volumes however they want. Also will likely need to set nodeSelector entries, and there's no good way to template or script that. So they'll deploy the toy first and then upgrade. Otherwise deployment is pretty gruesome.